bayes/src/main/java/com/enigmastation/ml/bayes/Feature.java

```diff
@@ -33,13 +42,19 @@ public void setCategories(Map<Object, Integer> categories) {
     }
 
     public Object getFeature() {
-        return feature;
+        return featureObject;
     }
 
     public void setFeature(Object feature) {
-        this.feature = feature;
+        this.featureObject = feature;
     }
 
+    /**
```

Comment:
This is a somewhat unnecessary javadoc comment. getCountForCategory(String category)... returns a count. For that category. It's not a BAD comment - I mean, hey, comments are great! - but the method name and parameter types and arguments are *enough*.